pkg/common/cns-lib/volume/manager.go

```diff
@@ -242,7 +242,25 @@ func GetManager(ctx context.Context, vc *cnsvsphere.VirtualCenter,
 	clusterFlavor cnstypes.CnsClusterFlavor) (Manager, error) {
 	log := logger.GetLogger(ctx)
 	managerInstanceLock.Lock()
-	defer managerInstanceLock.Unlock()
+	defer func() {
```

Comment:
Why this change? seems equivalent to previous

Reply:
https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/master/pkg/common/cns-lib/volume/manager.go#L262
few places like this, it returns the existing vc instance. updated code to be sure it always set version to vc client